shared/fixes/bash/accounts_authorized_local_users_sidadm_orasid.sh

```diff
@@ -7,31 +7,31 @@ default_os_user="root"
 
 # add users sidamd, orasid, sapadm and oracle if needed
 userlist="root"
+sapmnt_SID_stem="/sapmnt/[A-Z][A-Z0-9][A-Z0-9]"
```

Comment:
The regex is not identical to the one you carved out of the script - is that deliberate, to not have explicit beginning and ending there?

Reply:
@dahaic It shows the same result with and without the beginning and ending characters '^' and '$' in bash.  But you are correct, it's better to add these characters to keep consistent with OVAL pattern match convention.

Reply:
You are right, it's implicit in the case of `-regex` it seems, sorry, I could have checked it myself before commenting :) Anyway I think it's slightly more readable now, for those not familiar with the argument.